frontend/src/lib/api/types.ts

```diff
@@ -52,6 +52,7 @@ export interface Scratch extends TerseScratch {
     source_code: string
     context: string
     diff_label: string
+    claim_token: string | null
```

Comment:
This should be able to be undefined since ScratchSerializer doesn't include it

Reply:
Alternatively do `interface ClaimableScratch extends Scratch` like the backend

Reply:
Done